makerdao/governance-portal-v2
# Feature/sc 1091/allow executive card to open in new tab

### Link to Shortcut ticket:

https://app.shortcut.com/dux-makerdao/story/1091/allow-executive-card-to-open-in-new-tab

### What does this PR do?

Allow executive details and delegate detail buttons to be opened in a new tab by using `passHref` from `next/link` to a Theme UI link

Changes executive card from being clickable as a whole to having a "View Detail" button. This is more consistent with how the poll and delegate cards are implemented.

Also renames `InternalLink` to `ThemeUILink` for consistency throughout app

### Steps for testing:

Go to executive page, see that right-clicking one now allows you to open it in a new tab

See new detail button on exec cards

Same with delegate detail button

Screenshots:

Connected wallet:
<img width="819" alt="Screen Shot 2022-03-03 at 12 33 52 PM" src="https://user-images.githubusercontent.com/5225766/156560929-23698a28-ae7a-44c2-bbd6-5a26e75f7919.png">
<img width="445" alt="Screen Shot 2022-03-03 at 12 34 39 PM" src="https://user-images.githubusercontent.com/5225766/156560950-4fb335a9-20e3-48ae-8fd0-730332977a58.png">

No connected wallet:
<img width="817" alt="Screen Shot 2022-03-03 at 12 36 23 PM" src="https://user-images.githubusercontent.com/5225766/156560974-34f24a09-38e3-40eb-91da-e62e9b1bbadd.png">
<img width="466" alt="Screen Shot 2022-03-03 at 12 36 16 PM" src="https://user-images.githubusercontent.com/5225766/156560982-e6283b1f-6811-455a-bc9d-51c4c805d729.png">
